_modules/hubble.py

```diff
@@ -107,25 +103,27 @@ def audit(configs=None,
     if configs is None:
         return top(verbose=verbose,
                    show_success=show_success,
-                   show_compliance=show_compliance,
-                   show_profile=show_profile,
-                   debug=debug)
+                   show_compliance=show_compliance)
 
     if __salt__['config.get']('hubblestack:nova:autoload', True):
         load()
     if not __nova__:
         return False, 'No nova modules/data have been loaded.'
 
-    if verbose is None:
-        verbose = __salt__['config.get']('hubblestack:nova:verbose', False)
     if show_success is None:
         show_success = __salt__['config.get']('hubblestack:nova:show_success', True)
     if show_compliance is None:
         show_compliance = __salt__['config.get']('hubblestack:nova:show_compliance', True)
-    if show_profile is None:
-        show_profile = __salt__['config.get']('hubblestack:nova:show_profile', True)
-    if debug is None:
-        debug = __salt__['config.get']('hubblestack:nova:debug', False)
+    if show_profile is not None:
+        salt.utils.warn_until(
+            'Nitrogen',
+            'Keywork argument \'show_profile\' is no longer supported'
+        )
```

Comment:
I don't think we can actually properly use salt.utils.warn_until, since we don't match salt versions. Otherwise if someone installs a version of hubble with this warn_until on a Nitrogen version of salt, it will stacktrace. I think we just need to debug log the warning that it's deprecated but still leave it in for awhile.

(There are a bunch of other instances of this, I won't mark them individually)